requirements/edx/base.txt

```diff
@@ -258,7 +259,7 @@ django-config-models==2.3.0
     #   edx-enterprise
     #   edx-name-affirmation
     #   lti-consumer-xblock
-django-cors-headers==3.12.0
+django-cors-headers==3.13.0
```

Comment:
Added support for `Python 3.11` and `Django 4.1`. No breaking change in the [CHANGELOG](https://github.com/adamchainz/django-cors-headers/blob/main/HISTORY.rst#3130-2022-06-05).